zerver/tests/_trial_temp/test.log

```diff
@@ -0,0 +1 @@
+2020-05-28 17:42:38+0500 [-] Log opened.
```

Comment:
I think that you accidentally left this file in. Actually, the whole `_trial_temp` folder shouldn't exist.

Reply:
Oh, my bad!